paragon/_overrides.scss

```diff
@@ -196,7 +196,6 @@ select.form-control {
 }
 
 .container-fluid {
-  max-width: 1140px + 2 * $grid-gutter-width;
```

Comment:
Sanity check - this intentional?

Reply:
It is. This rule needs to be removed so we can all unify on the new container sizes in Paragon that manage max-widths. This rule is preventing them from working. So far it seems most teams have been making their own work arounds for max widths.